src/contentChunk.js

```diff
@@ -135,6 +135,10 @@ export const addBodiesToTransactions = async (transactions) => {
       transaction.orchestrations = await addBodiesToTransactions(transaction.orchestrations)
     }
 
+    if (transaction.routes && transaction.routes.length > 0) {
```

Comment:
With the statement above, the code breaks if the transaction.routes is true but not an array. I think we should check if "transaction.routes" is an array

> _Array.isArray( transaction.routes)_ 

>

Reply:
Please also fix the condition for the orchestrations above